src/components/DiagramFrame/utils.ts

```diff
@@ -40,3 +56,37 @@ export function buttonShade(exploreNameSel: string, currentExplore: ILookmlModel
   }
   return undefined
 }
+
+/**
+ * Prepares a list of diagrammable models for the Choose a Model dropdown
+ * @param unfilteredModels - the unprepared list of models
+ */
+export function prepareModelDropdown(unfilteredModels: ILookmlModel[]) {
+  if (unfilteredModels) {
```

Comment:
Method signature indicates that this is required parameter

could be simplified to something like
```
export function prepareModelDropdown(unfilteredModels: ILookmlModel[] = []) {
  return unfilteredModels.filter((d: ILookmlModel)=>{
    return d.explores.length >= 1 && !DIAGRAM_IGNORED_MODELS.includes(d.name)
  }).map((d: ILookmlModel)=>{
    return {
      value: d.name,
      label: d.label
    }
  }).sort((a: SelectOptionProps, b: SelectOptionProps) => a.label < b.label ? -1 : 1)
}
```